### PR TITLE
chore: introduce prow-github-actions

### PR DESCRIPTION
**Pull request by erikgb**
This is an introduction to https://github.com/jpmcb/prow-github-actions, which will allow us to use the most basic slash commands. We'll see how it works and eventually configure more features later.

### CODE REVIEW

File: .github/workflows/prow-commands.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Handle prow slash commands
```
Comment (bendikp) [inline .github/workflows/prow-commands.yml:1]:
```suggestion
---
name: Handle prow slash commands
```

**Comment by erikgb** [comment]
image

We will test on the next PR.

**Review by bendikp — APPROVED** [review]

**Pull request merged**